### ISSUE TITLE
Calendar show only 6 days in a row

### ISSUE THREAD

**Issue by mickeychu** [wakirin/Litepicker#17]
Hi @wakirin , sorry for bothering you again.

I just noticed that on my calendar, there's only 6 days in a row
![Screenshot 2019-12-24 at 16 02 48](https://user-images.githubusercontent.com/7807777/71416329-9a92fd00-2668-11ea-9fcf-39c814b2507e.png)

If I enable week number:

![Screenshot 2019-12-24 at 16 41 10](https://user-images.githubusercontent.com/7807777/71417112-4a1d9e80-266c-11ea-8115-c6a889f83e2c.png)

I'm using Bootstrap CSS. Should there be any conflicts?

Here's my options: 

```
var picker = new Litepicker({ 
                element: document.getElementById('litepicker'),
                format: 'DD.MM.YYYY',
                numberOfMonths: 2,
                numberOfColumns: 2,
                singleMode: false,
                showTooltip: false });
``` 

And the template:

```
<div class="row">
        <input type="text" id="litepicker">
</div>
```

Thank you for your support!!!

**Comment by wakirin** [comment]
Hi,
I thought it fixed (#11), but now I see the problem. 
You need to add this style:
```
.litepicker .container__months .month-item {
    box-sizing: content-box;
}
```

I will include this fix in next release.

**Comment by mickeychu** [comment]
Thanks @wakirin. Works like a charm!
